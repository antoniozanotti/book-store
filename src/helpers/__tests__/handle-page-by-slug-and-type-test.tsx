/**
 * @jest-environment node
 */
import { handlePageBySlugAndType } from "../handle-page-by-slug-and-type";
import { PageType } from "@/types/page-type";
import { productWithPagEandBookMock } from "@/domain/product/__tests__/product-with-page-and-book-mock";

describe("handlePageBySlugAndType", () => {
  it("should return title and description", async () => {
    const value = await handlePageBySlugAndType(productWithPagEandBookMock.page.slug, PageType.PRODUCT);
    expect(value).toMatchSnapshot();
  });
});