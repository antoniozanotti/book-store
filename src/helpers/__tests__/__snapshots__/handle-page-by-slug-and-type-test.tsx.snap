// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handlePageBySlugAndType should return title and description 1`] = `
{
  "description": "Canto eum versus somniculosus mollitia conforto celebrer iste. Surculus facere vix acceptus. Cohaero demo sufficio venio autus animi ex adversus quam.",
  "title": "Book The Four Agreements: A Practical Guide to Personal F...",
}
`;
